.env

```diff
@@ -140,3 +140,9 @@ MAINNET_SEND_MANY_CONTRACT_ID=SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.send-man
 # of openapi.yaml for test / development NODE_ENV. 
 # For production, /doc is not served if this env var is not provided.
 # API_DOCS_URL="https://docs.hiro.so/api" 
+
+# For use while syncing. Places the API into an "Initial Block Download(IBD)" mode, 
+# forcing it to stop any redundant processing until the node is fully synced up to its peers. 
+# Some examples of processing that are avoided are: 
+# REFRESH MATERIALIZED VIEW SQLs that are extremely CPU intensive on the PG instance, Mempool messages, etc.,
+# IBD_MODE_UNTIL_BLOCK=123
```

Comment:
Since the default is null, you should leave it as blank in the example (remove the 123)

Reply:
done